change-beta/@azure-communication-react-da2d5b43-20a8-4bc0-8cbe-09ceb376a06f.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "move client commits to separate branch",
```

Comment:
What is this change really doing. I dont think this message constructively adds any value to this commit. Perhaps it should be "adding raised hand functionality to stateful client"

Reply:
message type should also be pre-release